build/DependencyVersions.props

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <MicrosoftAspNetCoreAppPackageVersion>2.1.0-preview2-30338</MicrosoftAspNetCoreAppPackageVersion>
+    <MicrosoftAspNetCoreAppPackageVersion>2.1.0-preview2-28309</MicrosoftAspNetCoreAppPackageVersion>
```

Comment:
Why was this branch on dev?

Reply:
This is using something from our dev branch because this change was originally made to cli's master branch. #8810  Our build numbers are confusing b/c we have separate counters for the dev and release branches. This change makes it look like we actually went back in time, but the build is actually newer than 30338. I updated CI so the next release/2.1 builds should be build # 30400+ and dev will be 32000+.